src/types/Posts.ts

```diff
@@ -0,0 +1,11 @@
+import { Comments } from './Comments';
+import { User } from './User';
+
+export interface Posts {
```

Comment:
Post is a type of original Object from Posts.ts
and you create a new object with user & comments in it!